db/migrate/20170331194256_rename_users_recovery_code_to_personal_key.rb

```diff
@@ -0,0 +1,5 @@
+class RenameUsersRecoveryCodeToPersonalKey < ActiveRecord::Migration
+  def change
+    rename_column :users, :recovery_code, :personal_key
```

Comment:
Now that we're in more of a 'production' state, I expect we shouldn't just rename columns like this? Since we deploy to multiple app servers, we'll need to have +1/-1 compatibility? Like I'd expect a PR that adds a new column, then another PR that reads from either column?